### ISSUE TITLE
Proctoring MOM

### ISSUE THREAD

**Issue by Yogeshtake2** [taketwotech/docs-proctoring#2]
https://docs.google.com/document/d/1YKL2_ITO-xVohq2SVZ4DVuID-RzMVHeUKZFEibVKqZk/edit.

**Comment by Yogeshtake2** [comment]
meeting on 18-10-24 added

**Comment by PriyankaTake2** [comment]
For MoM dated 18-10-2024:

- Points 1, 2, and 3 need to be discussed in the next team meeting. @Yogeshtake2 please ensure these are addressed.
- Point 4 is in progress.
- Point 5 is awaited. @Yogeshtake2 any update from Ruchi? Please follow-up today and send a reminder.

**Comment by Yogeshtake2** [comment]
@PriyankaTake2 yes i got some points from ruchi yesterday in personal chat, i need to discuss those points with technical team.

**Comment by Yogeshtake2** [comment]
Added 21-10-24 MoM in the doc

**Comment by Yogeshtake2** [comment]
Added 22-10-24 MoM in the doc

**Comment by Yogeshtake2** [comment]
added 25-10-24 MoM in the doc
